pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -288,6 +302,8 @@ func (dswp *desiredStateOfWorldPopulator) processPodVolumes(pod *v1.Pod) {
 				volumeSpec.Name(),
 				uniquePodName,
 				err)
+
+			dswp.recordFailedProcessPodEvent(pod, fmt.Sprintf(errEventTemplate, podVolume.Name, format.Pod(pod), err))
```

Comment:
Should this one use the same msg as its log message?

Reply:
do we want to expose internal data structure like "desiredStateOfWorld" to end users in the pod event?

Reply:
Hm I guess not.  This is fine then.

Reply:
ok, I'll change this in the later update